jakarta.batch.arquillian.exec.example/README.md

```diff
@@ -0,0 +1,40 @@
+<!--- 
+Copyright (c) 2021 Contributors to the Eclipse Foundation
```

Comment:
I think the Jakarta process involves a careful review of the copyright.  I know ideally we do this before merging but if it's just me and you involved for now we can I'm sure resolve it before final release, if necessary.

Reply:
I'm not sure whether you suggest a change here or we can leave this as is for this PR. I followed the [Eclipse Project Handbook](https://www.eclipse.org/projects/handbook/#ip-copyright-headers), which states that "Alternatively, project teams can adopt a more generic copyright header for each file and include the identities of the copyright holders in the notice file."